### PR TITLE
Adds support Create and Delete of logical enclosures

### PR DESCRIPTION
**Pull request by Asserdna**

### CODE REVIEW

File: examples/logical_enclosures.py [hunk 1]
```diff
@@ -34,12 +34,49 @@
...
 }
 
-
 # Try load config from a file (if there is a config file)
 config = try_load_from_file(config)
 
 oneview_client = OneViewClient(config)
 
+try:
+    # The valid enclosure URIs need be inserted sorted by URI
```
Comment (abiliogp) [inline examples/logical_enclosures.py]: need _to_ be

File: examples/logical_enclosures.py [hunk 2]
```diff
@@ -34,12 +34,49 @@
...
+    # The number of enclosure URIs is conform enclosure count in enclosure group
```
Comment (GustavoHennig) [inline examples/logical_enclosures.py]: This sentence is not clear to me, I suggest something different:
"The number of enclosure URIs must be in accordance with enclosure count in the enclosure group"
or
"The number of enclosure URIs must be equal to the enclosure count in the enclosure group"

File: examples/logical_enclosures.py [hunk 3]
```diff
@@ -34,12 +34,49 @@
...
+    options = dict(
+        enclosureUris=[],
+        enclosureGroupUri="",
+        forceInstallFirmware=False,
+        name="LogicalEnclosure2"
+    )
+
+    # Get enclosure group for create logical enclosure
```
Comment (GustavoHennig) [inline examples/logical_enclosures.py]: Wouldn't "for creating" more appropriate here?

File: examples/logical_enclosures.py (hunk 2 above)
Comment (Asserdna) [inline examples/logical_enclosures.py]: I fixed it

File: examples/logical_enclosures.py (hunk 3 above)
Comment (Asserdna) [inline examples/logical_enclosures.py]: I fixed it

File: examples/logical_enclosures.py (hunk 1 above)
Comment (Asserdna) [inline examples/logical_enclosures.py]: I fixed it.

**Pull request merged**